database/StudentServiceMySql.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace database;
+
+use src\Student;
+
+class StudentServiceMySql implements StudentServiceInterface
+{
+	private $conn;
+
+	public function __construct($conn)
```

Comment:
tip: you can use Typehinting when passing parameters -> let the __constructor() function know what type is $conn and you will gain access to it's methods